tutorials/babble/drink/__init__.py

```diff
@@ -0,0 +1,2 @@
+from drink_model import DrinkModel
+from image_objects import BBox
```

Comment:
We shouldn't use this BBox object anywhere.

Reply:
removing stuff!